docs/maintainers/repository-structure.md

```diff
@@ -52,6 +52,7 @@ The following labels are used to categorize [issues in the v3 repository][github
 - [status/help-wanted][github-issues-status-help-wanted]: Extra attention is needed
 - [status/in-progress][github-issues-status-in-progress]: This issue is being worked on
 - [status/draft][github-issues-status-draft]: This issue or pull request is a draft, and not ready for review/merge.
+- [status/wontfix][github-issues-status-wontfix]: This issue or pull request is out of scope and will not be fixed. 
```

Comment:
As "won't fix" is two words, should it be:

```suggestion
- [status/wont-fix][github-issues-status-wontfix]: This issue or pull request is out of scope and will not be fixed. 
```

Reply:
`wontfix` is the canonical spelling of this label/status in the context of bug trackers